### PR TITLE
Update students.yml

### PR DESCRIPTION
**Pull request by ShivamRai2003**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -11,6 +11,13 @@
 #   lat: Student's Latitude
 #   lng: Student's Longitude
 
+ - name: Shivam Rai
+   country:India
```
Comment (nikit19) [inline _data/students.yml]: Please add a space after country:

File: _data/students.yml [hunk 2]
```diff
@@ -11,6 +11,13 @@
...
+   github: ShivamRai2003
+   image:https://avatars3.githubusercontent.com/u/34955127?s=460&v=4
```
Comment (nikit19) [inline _data/students.yml]: space after  image:
and you are supposed to write the name of the image that you have uploaded

File: _data/students.yml [hunk 3]
```diff
@@ -11,6 +11,13 @@
...
+   twitter: Shivam Rai
+ ShivamR44888706
```
Comment (nikit19) [inline _data/students.yml]: Remove this line -  ShivamR44888706

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make these changes and you have to upload the photo in the img/students folder

**Comment by ShivamRai2003** [comment]
where the img blogs found

**Comment by nikit19** [comment]
Sorry I meant the students folder.
There will be a folder named img. 
Open it.
Inside it there will be a folder named students.
Open it and upload your image.

**Comment by ShivamRai2003** [comment]
uploads are disabled it is saying

**Comment by ShivamRai2003** [comment]
Uploads are disabled.

File uploads require push access to this repository

**Comment by ShivamRai2003** [comment]
now what i do

**Comment by ShivamRai2003** [comment]
please merge it i want to approve my task

**Comment by nikit19** [comment]
You have to upload the file in your own forked repository

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Where is your image. pls add it

**Comment by codedsun** [comment]
@ShivamRai2003 Please add your image corresponding, and this can be done by copying to the following folder (/img/students) and then do the following changes in the commit by commit --amend.
Tell us if you get stuck!

**Comment by ShivamRai2003** [comment]
please send the video to do this i am not getting anything

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Please link your PR to the issue it is trying to solve by mentioning it under the description.

File: _data/students.yml [hunk 4]
```diff
@@ -11,6 +11,14 @@
 #   lat: Student's Latitude
 #   lng: Student's Longitude
 
+
+ - name: Shivam Rai
+   country: India
+   github: ShivamRai2003
+   image: (rai)shivam.jpg
```
Comment (anshumanv) [inline _data/students.yml:18]: Please upload an image by this name.

**Comment by ShivamRai2003** [comment]
nikit please see i have changed all
  https://github.com/ShivamRai2003/gci17.fossasia.org/blob/b168c5a2d3bb41b255721453dff3954e28574498/img/students/(rai)shivam.jpg
as you can see i have uploaded
  https://github.com/ShivamRai2003/gci17.fossasia.org/pull/6
as you can see that this is the correct image and i have sended the pull rquest also
  and added squash commit also

File: _data/students.yml (hunk 4 above)
Comment (ShivamRai2003) [reply _data/students.yml:18]: please upload it so i my task should be reviwed

Comment (ShivamRai2003) [reply _data/students.yml:18]: please update my profile to website

**Review by isuruAb — COMMENTED** [review]
hello @ShivamRai2003 ,I can't see the image in your PR.please upload your image and don't forget to squash your commits.

File: _data/students.yml (hunk 4 above)
Comment (ShivamRai2003) [reply _data/students.yml:18]: isuruAb where is the commits are available

Comment (ShivamRai2003) [reply _data/students.yml:18]: i have changed all

**Review by gabru-md — CHANGES_REQUESTED** [review]
Image is missing :)
Please add the image and squash your commits

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
> You'll have to learn about git.

Fork and then upload.
Then make a PR.
Or use git-cli

**Pull request closed without merging**